elastic/apm-server
# deprecate span destination name and type fields

## Checklist




## How to test these changes

Send some events to apm-server with span.destination.service.resource and without span.destination.service.name nor span.destination.service.type.
Events should be accepted.

## Related issues

Closes  https://github.com/elastic/apm-server/issues/5466
